docs/source/demo.rst

```diff
@@ -95,7 +95,7 @@ equivalent command using:
 .. code-block:: bash
 
     $ source ga4gh-env/bin/activate
-    (ga4gh-env) $ ga4gh_client readgroupsets-search --datasetIds XXXX http://localhost:8000/current
+    (ga4gh-env) $ ga4gh_client readgroupsets-search --dataset_ids XXXX http://localhost:8000/current
```

Comment:
In general I think command line arguments should be hyphenated instead of underscored... that seems more the UNIX way...